ledger/participant-state/kvutils/src/test/suite/scala/com/daml/ledger/participant/state/kvutils/committer/transaction/validation/ModelConformanceValidatorSpec.scala

```diff
@@ -188,9 +173,9 @@ class ModelConformanceValidatorSpec
           aTransactionEntry,
         )
       inside(step) { case StepStop(logEntry) =>
-        logEntry.getTransactionRejectionEntry.hasInconsistent shouldBe true
-        logEntry.getTransactionRejectionEntry.getInconsistent.getDetails should startWith(
-          "Missing input state for key contract_id: \"#inputContractId\""
+        logEntry.getTransactionRejectionEntry.getReasonCase shouldBe DamlTransactionRejectionEntry.ReasonCase.MISSING_INPUT_STATE
+        logEntry.getTransactionRejectionEntry.getMissingInputState.getKey.toString should startWith(
```

Comment:
I agree that it would make things easier and nicer to read, just tried to only "update" the tests by keeping them as they were as much as possible. Will try to fix this as well